src/service.ts

```diff
@@ -25,6 +26,13 @@ const rpc = new RpcProvider(message => {
 process.on('message', message => rpc.dispatch(message));
 
 const typescript: typeof ts = require(process.env.TYPESCRIPT_PATH!);
+const wrapperConfig: TypeScriptPatchConfig = {
```

Comment:
Isn't this more like `patchConfig`?

Reply:
Argh, yes. Remnant of an older #231 approach that still lurks my mind :laughing:  Gonna change it :)